data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/GetEvents.java

```diff
@@ -0,0 +1,115 @@
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.RES_OK;
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.TAG_EVENTS;
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.TAG_VALIDATOR_REQUIRED;
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.TOPICS;
+import static tech.pegasys.teku.beaconrestapi.handlers.v1.EventSubscriptionManager.VALID_EVENT_TYPES;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.javalin.http.Context;
+import io.javalin.http.Handler;
+import io.javalin.http.sse.SseClient;
+import io.javalin.http.sse.SseHandler;
+import io.javalin.plugin.openapi.annotations.HttpMethod;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiContent;
+import io.javalin.plugin.openapi.annotations.OpenApiParam;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jetbrains.annotations.NotNull;
+import tech.pegasys.teku.api.ChainDataProvider;
+import tech.pegasys.teku.api.DataProvider;
+import tech.pegasys.teku.beaconrestapi.ListQueryParameterUtils;
+import tech.pegasys.teku.beaconrestapi.schema.BadRequest;
+import tech.pegasys.teku.infrastructure.events.EventChannels;
+import tech.pegasys.teku.provider.JsonProvider;
+
+public class GetEvents implements Handler {
+  private static final Logger LOG = LogManager.getLogger();
+  public static final String ROUTE = "/eth/v1/events";
+  private final JsonProvider jsonProvider;
+  private final EventSubscriptionManager eventService;
+
+  public GetEvents(
+      final DataProvider dataProvider,
+      final JsonProvider jsonProvider,
+      final EventChannels eventChannels) {
+    this(dataProvider.getChainDataProvider(), jsonProvider, eventChannels);
+  }
+
+  GetEvents(
+      final ChainDataProvider provider,
+      final JsonProvider jsonProvider,
+      final EventChannels eventChannels) {
+    this.jsonProvider = jsonProvider;
+    eventService = new EventSubscriptionManager(provider, jsonProvider, eventChannels);
+  }
+
+  @OpenApi(
+      path = ROUTE,
+      method = HttpMethod.GET,
+      summary = "Subscribe to node events",
+      tags = {TAG_EVENTS, TAG_VALIDATOR_REQUIRED},
+      description =
+          "Provides endpoint to subscribe to beacon node Server-Sent-Events stream. Consumers should use"
+              + " [eventsource](https://html.spec.whatwg.org/multipage/server-sent-events.html#the-eventsource-interface)"
+              + " implementation to listen on those events.",
+      queryParams = {
+        @OpenApiParam(
+            name = TOPICS,
+            required = true,
+            description =
+                "Event types to subscribe to."
+                    + "Available values : head, block, attestation, voluntary_exit, finalized_checkpoint, chain_reorg"),
```

Comment:
Looks like there's an extra space after the ":"
```suggestion
                    + "Available values: head, block, attestation, voluntary_exit, finalized_checkpoint, chain_reorg"),
```

Reply:
copy / paste from the spec :) fixed